docs/cli/cmdline/script/info.md

```diff
@@ -0,0 +1,49 @@
+---
+title: terramate script info - Command
+description: The script info command gives you information about a script
+
+prev:
+  text: 'Run'
+  link: '/cli/cmdline/run'
+
+next:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+---
+
+# Script Info
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script info` command shows details of all scripts matching the command line argument. The details include:
+
+| Name        | Meaning                                |
+| ----------- | -------------------------------------- |
+| Definition  | Where the script is defined            |
+| Description | The description label in the script    |
+| Stacks      | The stacks where the script can be run |
+| Jobs        | The commands that comprise the script  |
+
+This information is always relative to the current directory (or the value of `-C`).
+
+**Note: ** Scripts with the same name that are overwritten in child directories are considered separate scripts.
```

Comment:
```suggestion
**Note: ** Scripts with the same name that are overridden in child directories are considered separate scripts.
```
reasoning: overwritten usually means mutation but in this case, they are just overridden/shadowed.

Reply:
in that case I would say `overridden (shadowed)` to be really explicit (e.g. I understand shadowed but never heart overridden)

Reply:
shadowed seems wrong, i would go with @i4ki suggestion here.

@soerenmartius this is a common concept even in terraform https://developer.hashicorp.com/terraform/language/files/override